### DISCUSSION TITLE
blog/2023/07/28/the-problem-with-ai-trolley-dilemma/

### DISCUSSION THREAD

**Discussion by giscus** [InfiniBrains/Awesome-GameDev-Resources#31] [General]
# blog/2023/07/28/the-problem-with-ai-trolley-dilemma/

Awesome GameDev Resources for learning how to make games

https://courses.tolstenko.net/blog/2023/07/28/the-problem-with-ai-trolley-dilemma/

**Comment by AlexRamosBrazil** [comment]
You're assuming that the AI is a program that is made to execute only commands that were defined by a programmer. In this sense your text can be considered right, however AI is not only that, it "learns" with experience, so maybe in the first time it will kills the passengers, on the second one, the pedestrians, on the third one, it will try to reduce speed and in the next one throw the car in the cliff at the road side... Just joking man, love your text. :P

**Reply by tolstenko** [reply]
Thanks for the comment. Yeah... I guess the problem is the premise, I do not contest the outcome of the dilemma, I challenge the existence of it.
